content/next/index.md

```diff
@@ -187,6 +187,13 @@ revert: let us never again speak of the noodle incident
 Refs: 676104e, a215868
 ```
 
+### What writing form should I use?
+
+We recommend writing a commit description and body using the [imperative](https://en.wikipedia.org/wiki/Imperative_mood) present tense writing form. There are a significant number of examples of this writing form for commits: [1](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)[2](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#subject)[3](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project)[4](https://medium.com/@danielfeelfine/commit-verbs-101-why-i-like-to-use-this-and-why-you-should-also-like-it-d3ed2689ef70)[5](https://chris.beams.io/posts/git-commit/#imperative)
+
+If you are using a verb, such as "update" and "revert" as a type, you may see it as the verb of the imperative phrase only if the change is really trivial. For more complex things such as "fix", you should stick to describing what exactly you have done using the imperative form. Writing things like _fix: crash on login_ is a huge no-no.
```

Comment:
Do not really agree with it `fix: crash on login`. Because it may have a description to explain more details.

Could you please elaborate more?

Reply:
I can rephrase it to "Writing things like... without further description in the commit body is a huge no-no. You should at least try to mention the root cause."

The story is that I got fed up with someone else writing `fix: fooBarProcess jams up console` when `fix: turn off console logging for fooBarProcess` works much better. Crashes and more complex bugs are indeed harder to describe in 44 characters, although people should still try to write stuff like `fix: free up fooObj to avoid instant OOM death` in my opinion. The "huge" part really needs some re-considering.

Reply:
Any update on this @Artoria2e5 ?

Reply:
The imperative style is quite well established good practice. See for example https://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project

Reply:
I do aggree with imperative style, but I don't aggree with this specific example.

The commit-message, especially in the context of conventional-commit, should be a user-oriented changelog. And a user shouldn't have to care nor understand the actual cause of the issue. We only tell the user what is fixed, not the internal details that caused the bug. Ofc, the message should contain under which conditions the bug was appearing, exaample: `fix: OOM when calling fooObj`

The body of the commit message may eventually contains some more in-depth/developer-oriented explanation of what was the root cause how it is fixed.